subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -509,10 +508,6 @@ int aws_fota_init(struct mqtt_client *const client,
 		return -EINVAL;
 	}
 
-	if (strlen(app_version) >= CONFIG_AWS_FOTA_VERSION_STRING_MAX_LEN) {
```

Comment:
This check is no longer needed?

Reply:
`app_version` is removed so yes.

Reply:
Bit seems CONFIG_AWS_FOTA_VERSION_STRING_MAX_LEN also should be removed I'll remove it.

Reply:
It was removed my bad :) is removed from kconfig AWS_FOTA_VERSION_STRING_MAX_LEN